docs/configuration/configuration.md

```diff
@@ -359,9 +359,9 @@ services:
 tags:
   [ - <string> ]
 
-# Node metadata used to filter nodes for a given service.
+# Node metadata pairs used to filter nodes for a given service.
```

Comment:
"names and values" maybe?

Reply:
Here is what consul says:

```
Metadata to filter nodes with the given key=value pairs. This flag may be specified multiple times to filter on multiple sources of metadata.
```

Thet do not speak about names but keys/values.

Reply:
Should I just put

```
Metadata to filter nodes with the given key=value pairs for a given service
```

as description?

Reply:
Okay for that bit, but the grammar feels off. How about:

Node metadata key/value pairs to filter nodes for a given service

Reply:
fixed